source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,51 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-04-21
+
+### Highlights
+
+#### Apps Framework, lightweight interactive add-ons to Mattermost
+ - 
```

Comment:
Should we add something similar to what we did in mobile?
```
This feature is available on Mobile Apps when Apps Framework becomes available on Mattermost v5.35.0 and the proxy plugin is loaded on an instance. The launch for the Developer Preview of the Apps Framework is scheduled for April 29th.
```
Probably will need some rephrasing for this context. @aaronrothschild what do you think?

Reply:
I added a description.